index.js

```diff
@@ -63,7 +63,7 @@ module.exports = fp(fastifyRequestContext, {
 })
 module.exports.default = fastifyRequestContext
 module.exports.fastifyRequestContext = fastifyRequestContext
-
+module.exports.asyncLocalStorage = asyncLocalStorage
```

Comment:
can you add documentation with example usage (can be taken from your PR description)?